packages/xo-web/src/xo-app/host/tab-advanced.js

```diff
@@ -222,7 +226,12 @@ export default class extends Component {
                 </tr>
                 <tr>
                   <th>{_('hostIscsiName')}</th>
-                  <Copiable tagName='td'>{host.iSCSI_name}</Copiable>
+                  <td>
+                    <Text
+                      value={iscsiName === null ? '' : iscsiName}
```

Comment:
In which case `iscsiName` can be `null` ?

Reply:
when user clear it from the input.

Reply:
When a user clear an input of type text, the value will be an empty string. 
But its not significant here because its value is fetched from the server.